fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -137,24 +136,24 @@ def fit(self, X, y, aux_data=None, **kwargs):
         # Prep the quality metric
         self.quality_metric.set_data(X, y_vector, A)
 
-        if isinstance(self.disparity_metric, ConditionalOpportunity):
+        if isinstance(self.disparity_metric, ClassificationMoment):
             # We have a classification problem
             # Need to make sure that y is binary (for now)
             unique_labels = np.unique(y_vector)
             if not set(unique_labels).issubset({0, 1}):
                 raise RuntimeError(self._MESSAGE_Y_NOT_BINARY)
 
         # Prep the disparity metric and objective
-        self.disparity_metric.init(X, A, y_vector)
+        self.disparity_metric.load_data(X, y_vector, **kwargs)    # sensitive_features=A)
```

Comment:
Is there something you planned to do here? There's a comment at the end

Reply:
There are these in a few places. I think they're just conversion leftovers, and should just be removed.

Reply:
good catch. removing.